wiki/markdown/query.md

```diff
@@ -2,24 +2,24 @@
 
 |Method|Endpoint             |
```

Comment:
Contents of this file should be moved to query-aggregated-objects.md

Reply:
DONE

Reply:
Don´t see that it is done?

Reply:
Becuse it is not committed yet.

Reply:
Please write "Fixed in upcoming commit." next time :wink: